rskj-core/src/test/java/co/rsk/vm/VmDslTest.java

```diff
@@ -47,28 +54,28 @@ void invokeRecursiveContractsUsing400Levels() throws FileNotFoundException, DslP
 
         Block block = world.getBlockByName("b02");
 
-        Assertions.assertNotNull(block);
+        assertNotNull(block);
```

Comment:
I personally prefer keeping `Assertions.assertNotNull(<value>);` for consistency and clarity, as it clearly indicates the assertion library in use and helps avoid confusion, especially if multiple libraries are involved. What do you think about sticking with the explicit `Assertions` reference?

Reply:
Well, for me it makes more noise than help. Because basically all different Assertions library have the class Assertions, so I would sill need to go to definition and see which one is used no? Or is there a very different name recognisable only by the `Assertions.` ?

Reply:
I see your point. My concern was mainly about clarity and avoiding confusion when multiple libraries are involved. I personally prefer using `Assertions.<method>` to ensure that clarity, but if you prefer to keep it as is, that’s totally fine!